_data/toc/cloud-guide.yml

```diff
@@ -74,47 +71,50 @@ pages:
         - label: GitHub integration
           url: /cloud/integrations/github-integration.html
 
-    - label: Local environment setup
+    - label: Local development
       url: /cloud/access-acct/first-time-setup.html
       children:
-        - label: Prepare for local environment setup
-          url: /cloud/before/before-workspace.html
+        - label: Docker development
+          url: /cloud/docker/docker-development.html
+          include_versions: ["2.1","2.2","2.3"]
```

Comment:
If the menu item is supposed to be for every version instead of 2.0, i suggest using `exclude_versions: "2.0"`.

Reply:
Yes! I need to update with new TOC strategy! Thanks.